src/AutoRest.CSharp/Common/AutoRest/Plugins/NewProjectScaffolding.cs

```diff
@@ -292,7 +294,13 @@ private string GetSrcCsproj()
   </PropertyGroup>
 
   <ItemGroup>
-    <Compile Include=""$(AzureCoreSharedSources)AzureResourceProviderNamespaceAttribute.cs"" LinkBase=""Shared/Core"" />
+    <Compile Include=""$(AzureCoreSharedSources)AzureResourceProviderNamespaceAttribute.cs"" LinkBase=""Shared/Core"" />";
+            if (_needAzureKeyAuth)
```

Comment:
You actually can define `srcBrandedCsprojContent` as `@$"xxxx"`, e.g. multiple line formattable string. So there the codes can be like:
```suggestion
           { (_needAzureKeyAuth ? @"....." : @"....")}
```

Reply:
hello @archerzz  I have thought about this. but it cannot work for some encoding issue. I remember. So we need to maintain as it is.